notebooks/03-inference-rest.ipynb

```diff
@@ -11,7 +11,7 @@
     "import json\n",
     "import numpy as np\n",
     "\n",
-    "URL = \"http://modelmesh-serving.stock.svc.cluster.local:8008/v2/models/stock/infer\"\n",
+    "URL = \"http://modelmesh-serving:8008/v2/models/stocks/infer\"\n",
```

Comment:
I changed these to match the InferenceService name, but it's actually broken when it first gets deployed since it tries to download a model that hasn't been created yet.